packages/yoga/src/Stepper/web/Stepper.jsx

```diff
@@ -58,11 +63,14 @@ Stepper.propTypes = {
   /** Controls the active step, it receive the index value for showing some
    * step. Starting from 0. */
   activeStep: limitChildren,
+  /** Must be a color from yoga colors. */
+  secondary: bool,
```

Comment:
Exactly as described in the prop comment ::)

Reply:
I was talking with @matheushdsbr about this option, the initial approach was to receive a string that could be any yoga color, but after we talk about we decided to keep the component more unflexible. What do you think about it @matheushdsbr ?